packages/components/src/components/dropdown/dropdown.scss

```diff
@@ -273,7 +273,8 @@
         min-width: 15rem;
         width: 100%;
 
-        &:not(.cfd-personal-details-modal__form *):not(.trade-container__multiplier-dropdown):not(.dc-dropdown--left):not(.contract-type-info__dropdown) {
+        &:not(.cfd-personal-details-modal__form *):not(.trade-container__multiplier-dropdown):not(
+                .dc-dropdown--left):not(.contract-type-info__dropdown) {
```

Comment:
I believe this is not intended. Could you reset this file to `master`?

Reply:
Yep this is coming from the eslint configuration, updated 👍